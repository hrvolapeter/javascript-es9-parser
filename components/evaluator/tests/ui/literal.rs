var a = 1;
var b = true;
var c = "string";
var d = 'string';
var e = null;
console.log(a);
console.log(b);
console.log(c);
console.log(d);
console.log(e);
