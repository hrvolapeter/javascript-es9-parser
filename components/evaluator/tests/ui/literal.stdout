1
true
"string"
"string"
null
